sonic-xcvrd/scripts/xcvrd

```diff
@@ -631,11 +675,16 @@ class sfp_state_update_task:
         while not stopping_event.is_set():
             status, port_dict = platform_sfputil.get_transceiver_change_event()
             if status:
+                # Receive event on single SFP
                 for key, value in port_dict.iteritems():
+                    if key == EVENT_ON_ALL_SFP:
+                        logger.log_error("Receive event with unexpected content, key: {}, value: {}".format(key, value))
```

Comment:
>Receive [](start = 42, length = 7)

Past tense is better. As original 'Got'. Many time below.

Reply:
fixed.